README.md

```diff
@@ -4,6 +4,16 @@
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.2600909.svg)](https://doi.org/10.5281/zenodo.2600909)
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/volkamerlab/TeachOpenCADD/master)
 
+> ⚠ You are looking at a new release of the TeachOpenCADD repository with [major changes](https://github.com/volkamerlab/teachopencadd/pull/29).  
```

Comment:
I was wondering here to were we want to point? because after merging the changes are in? maybe rather link to the new version? tbd

Reply:
Good idea. We are creating a new release tag after we did the `packaging` > `master` merge, correct @jaimergp? 
If that is the case, we could keep the link as-is in this PR and update the link when we are on master.

Reply:
Same goes for the link to our soon-to-be conda package (to be added to one of the disclaimer's bullet points):

Reply:
(I added both to this PR's TODOs - to be moved around PRs as needed.)